src/Inter/interpreter.rs

```
@@ -1491,6 +1523,14 @@ impl Interpreter {
         Ok(())
     }
 
+
+    fn evaluate_enum_declaration(&mut self, identifier: &String, variants: &Vec<String>) -> Result<(), CPSError> {
```

Clippy (1.97.1)

[warning] the `Err`-variant returned from this function is very large

(warning)